### ISSUE TITLE
empty render tree shows before rendering

### ISSUE THREAD

**Issue by sritchie** [ChristopherChudzicki/mathbox-react#204]
mathbox-react prints a render tree for each `Mathbox`, but the render tree is currently always empty. I feel like this did work!

For example, on the current Cartesian story in the project's storybook:

<img width="587" alt="image" src="https://user-images.githubusercontent.com/69635/209706287-441d5b83-9f84-4bc5-a03e-10f309562d77.png">

**Comment by ChristopherChudzicki** [comment]
I need to think about this a bit more. I entirely understand the expectation that something like

```ts
const AxisDemo: React.FC = () => {
  return (
    <MB.ContainedMathbox containerStyle={{ height: "500px" }}>
      <MB.Cartesian>
        <MB.Axis color={new THREE.Color(0xff4136)} />
        <MB.Axis axis={2} color={new THREE.Color(0x2ecc40)} />
        <MB.Axis axis={3} color={new THREE.Color(0x0074d9)} />
      </MB.Cartesian>
    </MB.ContainedMathbox>
  )
}
```
would log a pretty Mathbox tree in the console. 

Some facts:
- the logging is coming from MathBox *From [here](https://github.com/unconed/mathbox/blob/master/src/index.js#L150)*
- mathbox-react is rendering mathbox components asynchronously (and, as far as I understand with react, it really needs to: "side effects" need to go in `useEffect` hooks, which are potentially asynchronous)

Additional consideration:
- a goal of mathbox-react is to enable much more dynamic visualizations than vanilla mathbox. Think: adding and removing objects from a mathbox scene, a la math3d.org 

In that more dynamic world, what should be printed? The initial scene? (That's what currently happens, but it's empty). The scene a few seconds later? The scene when nothing has changed within some time period? I dunno.

More useful, IMO, is assigning
```ts
window.mathbox = mathbox
```
so you can call `mathbox.inspect()` in the dev console yourself.

**Comment by sritchie** [comment]
Yeah I agree that this is a good use for ref, so you can inspect yourself whenever you want. I implemented something similar to Mathbox-react with jsxgraph.cljs, and I think the only natural time is basically after an initial render of the top level component, in the hook version of the first call to componentDidUpdate.

But maybe we should just remove the empty root rendering and document the inspect call.
